concurrency/Control/Concurrent/Classy.hs

```diff
@@ -33,10 +33,13 @@ module Control.Concurrent.Classy
   ) where
 
 import           Control.Concurrent.Classy.Chan
+import           Control.Concurrent.Classy.BoundedChan
```

Comment:
The build is failing because these modules are imported here, but not re-exported.  Did you mean to re-export them?